lib/_http_common.js

```diff
@@ -268,7 +268,6 @@ module.exports = {
   _checkIsHttpToken: checkIsHttpToken,
   chunkExpression: /(?:^|\W)chunked(?:$|\W)/i,
   continueExpression: /(?:^|\W)100-continue(?:$|\W)/i,
-  CRLF: '\r\n',
```

Comment:
Given that this module is semi-public, we shouldn't remove this property here without a proper deprecation.

Reply:
```suggestion
  CRLF: '\r\n', // TODO: Deprecate this.
```